### ISSUE TITLE
Keyframe false positive in DuplicateRoot

### ISSUE THREAD

**Issue by nschonni** [brigade/scss-lint#138]
Looks like I missed another edge case

``` scss
@keyframes slideouttoleft {
    from {
        transform: translateX(0);
    }

    to {
        transform: translateX(-100%);
    }
}

@keyframes slideouttoright {
    from {
        transform: translateX(0);
    }

    to {
        transform: translateX(100%);
    }
}
```

This flags `from` and `to` as duplicates
